app/assets/javascripts/change_current_provider.coffee

```diff
@@ -106,7 +113,7 @@ $(document).ready ->
         # Click the link that the user needs
         if linkType == 'delete-collection' && $('.collection-granule-count').text() != 'Granules (0)'
           $('#display-granules-modal').click()
-        else 
+        else
```

Comment:
Fixing these trailing blanks makes the PR harder to review. I'm not sure that the benefit justifies the confusion.  Just a thought.